pkgs/tools/text/shfmt/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "shfmt";
-  version = "3.9.0";
+  version = "3.10.0";
 
   src = fetchFromGitHub {
     owner = "mvdan";
     repo = "sh";
     rev = "v${version}";
```

Comment:
```suggestion
    rev = "refs/tags/v${version}";
```
While we're manually updating.

Reply:
No nitpicks, thanks.

Reply:
We should just do a treewide with this

Reply:
Either a treewide or get nix-update, nixpkgs-update, etc to add it.